src/mpid/ch4/src/ch4r_win.h

```diff
@@ -15,6 +15,7 @@
 #include "ch4i_util.h"
 #include <opa_primitives.h>
 #include "mpir_info.h"
+#include "ch4r_symheap.h"
```

Comment:
This line change should not be placed int this commit, right? This commit is for separating function calls under different ch4-direct mode.

Reply:
I rebased this PR on top of #3212 for compiling & testing. This change should be a part of #3212. Please see my explanation in that PR.